openstack/blockstorage/v3/volumes/requests.go

```diff
@@ -122,6 +122,36 @@ func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pa
 	})
 }
 
```

Comment:
Same comments as above of course, one other thing to consider is do we need this in V2 or just V3?  I realize there's no shortage of folks that have V2 deployed and will be using V2 but before we start doing 2x changes for every add we might want to think about a strategy/policy here.

I'm hoping to get something moving again on the non-versioned objects approach that would help with this.

Reply:
> before we start doing 2x changes for every add we might want to think about a strategy/policy here.

This is the downside of doing the v2/v3 copy when Gophercloud's v2 implementation wasn't complete.

I think it is totally acceptable for someone to submit a v3 patch without the corresponding v2 patch. If this happens, I am happy to do the backporting myself. 

I realize there's a chance that the backported v2 feature may never be used. But if it helps one person down the road, it was worth the few extra steps.

Of course, I completely agree that the situation we're in is not the most efficient. At the moment, I'm personally in a position where manual maintenance is easier for me to handle than digging into a better long-term solution. I hope that changes soon, but this is the best way for me to be able to balance getting people the features they need in Gophercloud.

If you or anyone else is able to get a solution together which resolves this, I'd gladly make it a priority to review and a priority to drown the person in praise :)